Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Node.cpp

```diff
@@ -52,12 +52,8 @@ namespace NodeCpp
         // This change resolves this error by setting the version higher than any previous version.
         FixedVersioningIssue = 14,
 
-#if defined(CARBONATED)
```

Comment:
In LY the node version is 12, which is explicitly set in the code. So we set O3DE's to 12 too on June 20, 2023 to avoid error messages and probably conversion issues.  Default node version in O3DE was actually 2 that time, 14 was inserted later. It looks like they have a valid conversion code piece now, it makes sense to try removing the patch.